soft alert phrases

shakerrdev opened the pull request:
added soft alert phrases functionality.
example:
soft alert phrase: nsf
will trigger for nsf but not for nsfw

#38

shakerrdev committed:
auto formatter

shakerrdev committed:
added custom soft alert phrases

added soft alert phrases functionality.
example:
soft alert phrase: nsf
will trigger for nsf but not for nsfw

shakerrdev committed:
removed auto formatting

I think this is the correct formatting again now.

vyneer commented on dgg-utilities.user.js:
```diff
@@ -1514,1215 +1431,988 @@ function injectScript() {
-        ) {
-            let text = textarea.value.toLowerCase();
-            let resultCustom;
-            let resultCustomSoft;
```
Seems like you removed this line (```let resultCustomSoft;```) on accident while cleaning up, gotta add it back (leads to errors if array is empty) NODDERS

shakerrdev replied:
To remove the auto formatting I took your base version and cv pasted my stuff back into it and I guess I forgor that line nathanDerp

vyneer commented on dgg-utilities.user.js:
```diff
@@ -130,7 +130,9 @@ const configItems = {
   customPhrases     : new ConfigItem("customPhrases",      []      ),
+  customPhrasesSoft : new ConfigItem("customPhrasesSoft",  []      ),
   customColor       : new ConfigItem("customColor",        "1f0000"),
+  customSoftColor   : new ConfigItem("customSoftColor",    "680030"),
```
Maybe a more muted default color? Just so it's more in line with the others, y'know PepoComfy 
Something like ```#260019``` maybe?

shakerrdev committed:
re-added missing line; changed color

- re-added variable declaration that went missing while removing auto formatting
- changed customSoftColor default value

vyneer reviewed (approved):
LGTM!

vyneer commented:
Thank you very much! <3
